src/getMessageRequests.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
```

Comment:
It looks like this function is a copy/paste of getThreadList with `inbox` changed to `pending`.  Is there a reason we can't just add an optional boolean to getThreadList to select pending threads?